pkg/shell/templates/auth-failed.html

```diff
@@ -0,0 +1,31 @@
+<div class="modal-header">
+    <h4 class="modal-title" translatable="yes">Authentication Failed</h4>
+</div>
+<div class="modal-body">
+    <div translatable="yes">
+    {{#supported}}
+    <div translatable="yes">
```

Comment:
Unfortunately we cannot translate things with tags in them. I think the tags get killed.